benchmark/server.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const jstp = require('..');
+
+const maxConnections = +process.argv[2];
+
+const app = new jstp.Application('app', {
+  iface: {
+    method(connection, argument, callback) {
+      callback(null);
+    },
+  },
+});
+
+const server = jstp.net.createServer([app]);
+server.maxConnections = maxConnections;
+
+const socket = '/tmp/jstp_benchmark_ipc';
```

Comment:
This won't work on Windows at all (where named pipes are created inside a special namespace in the VFS), and maybe on some Unix flavors too (it isn't correct to rely on `/tmp/` being available, that's what `os.tmpdir()` is for). I am okay with landing this as is now, because the benchmark is not exposed to end users, but it should be fixed later. Aside from that, a hardcoded filename is prone to race conditions.